public/js/application.js

```diff
@@ -0,0 +1,65 @@
+function login() {
```

Comment:
You should not define global functions. Always put your code under a common namespace like `app`